### DISCUSSION TITLE
Getting started with fossunited.org development

### DISCUSSION THREAD

**Discussion by arunppsg** [fossunited/fossunited#576] [Q&A] [answered]
I am new to frappe and I am interested in contributing to the development of fossunited.org website and dashboard. To this end, I was able to setup frappe, start the site locally and access the dashboard.

In test.localhost:8000, I have:
<img width="1335" alt="Screenshot 2024-08-25 at 4 40 30 PM" src="https://github.com/user-attachments/assets/44c1b1b1-5595-48b5-ba1d-7ccbdb2a2f0c">

and in test.localhost:8080, I have: 
<img width="836" alt="Screenshot 2024-08-25 at 4 41 11 PM" src="https://github.com/user-attachments/assets/9ebe2ed2-e53d-416e-a0ee-6c44f3e03e09">

Now, I would like to figure out
- how to access [fossunited.org](https://fossunited.org/) dashboard locally
- add a user, chapter and a meetup in my local instance.

Any help on the above will be greatly appreciated.

**Comment by RiyaMathew-11** [comment, marked as answer]
The relevant doctypes you need here are: 

1. The `User` doctype - create a new user, and you can see, with the workflows added, a FOSS User Profile is also created along. (Check `FOSS User Profile` doctype)
2. The `FOSS Chapter` doctype - add a new chapter from here. 
3. The `FOSS Chapter Event` doctype - create a new event here. 

You can find the web views created along. 

PS: You can search for the whole list of Doctypes in the awesomebar - or find the quick link to all Doctypes by clicking the Build tab on the left.

**Comment by arunppsg** [comment]
Thanks, I was able to get the setup working - one quick question: how do I access [fossunited.org](fossunited.org) website locally? (the front landing page which one sees when they go to fossunited.org). I tried `yarn dev` inside the dashboard directory but it doesn't seem to work.

**Reply by RiyaMathew-11** [reply]
> Thanks, I was able to get the setup working - one quick question: how do I access [fossunited.org](fossunited.org) website locally? (the front landing page which one sees when they go to fossunited.org). I tried `yarn dev` inside the dashboard directory but it doesn't seem to work.

The public pages like that of [fossunited.org/home](https://fossunited.org), aren't made in Vue. They are built with normal HTML/CSS/JS as web templates. A webpage is built together by putting these templates together, with frappe's page builder feature for building webpages. 

You can create a webpage from Frappe desk, add route, and pull in the templates together. 

For example - the homepage is built using these templates. 

![Screenshot 2024-09-03 at 4 50 33 PM](https://github.com/user-attachments/assets/7630929a-4065-47bf-bfe9-c1df08027281)

The templates can be found [here](https://github.com/fossunited/fossunited/tree/develop/fossunited/fossunited/web_template) - The Web Templates list in your local desk setup defines them.
